odh-dashboard/base/deployment.yaml

```diff
@@ -14,6 +14,31 @@ spec:
     spec:
       serviceAccount: odh-dashboard
       containers:
+      - name: oauth-proxy
+        args:
+        - --https-address=:8443
+        - --provider=openshift
+        - --openshift-service-account=odh-dashboard
+        - --upstream=http://localhost:8080
+        - --tls-cert=/etc/tls/private/tls.crt
+        - --tls-key=/etc/tls/private/tls.key
+        - --cookie-secret=SECRET
```

Comment:
Do we need to parameterize te SECRET?

Reply:
likely, yes. I could use a pointer to an example for how to do that.

Downstream deployments will use a slightly different method to generate a random value of this (see https://github.com/red-hat-data-services/odh-deployer/blob/main/deploy.sh#L34)

Reply:
Hmm...we do not have a way to generate a random value at the moment.

Here is an example of a parameter:
- https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml#L11
- https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/params.env#L2
- For non-standard fields you need to add kind and path here: https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/params.yaml
- https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/kustomization.yaml#L43-L49 (the CM `parameters` is generated based on `params.env`)